core/db/migrate/20141101231208_fix_adjustment_order_presence.rb

```diff
@@ -5,7 +5,7 @@ def change
       adjustable = adjustment.adjustable
       if adjustable.is_a? Spree::Order
         adjustment.update_attributes!(order_id: adjustable.id)
-      else
+      elsif adjustable && adjustable.order
         adjustment.update_attributes!(adjustable: adjustable.order)
```

Comment:
@gmacdougall @JDutil The more I look at this, the more confused I am at this particular line.

This migration looks for adjustments without orders, but here it's not actually setting the `order`, it's setting the `adjustable`! Could that be a bug?

I'd expect to see `adjustment.update_attributes!(order: adjustable.order)`